doc/stomp_planner/stomp_planner_tutorial.rst

```diff
@@ -0,0 +1,221 @@
+
+    stomp/manipulator_rail:
+      group_name: panda_arm
+      optimization:
+        num_timesteps: 60
+        num_iterations: 40
+        num_iterations_after_valid: 0    
+        num_rollouts: 30
+        max_rollouts: 30 
+        initialization_method: 1 #[1 : LINEAR_INTERPOLATION, 2 : CUBIC_POLYNOMIAL, 3 : MININUM_CONTROL_COST
+        control_cost_weight: 0.0
+      task:
+        noise_generator:
+          - class: stomp_moveit/NormalDistributionSampling
+            stddev: [0.05, 0.8, 1.0, 0.8, 0.4, 0.4, 0.4]
+        cost_functions:
+          - class: stomp_moveit/CollisionCheck
+            collision_penalty: 1.0
+            cost_weight: 1.0
+            kernel_window_percentage: 0.2
+            longest_valid_joint_move: 0.05 
+        noisy_filters:
+          - class: stomp_moveit/JointLimits
+            lock_start: True
+            lock_goal: True
+          - class: stomp_moveit/MultiTrajectoryVisualization
+            line_width: 0.02
+            rgb: [255, 255, 0]
+            marker_array_topic: stomp_trajectories
+            marker_namespace: noisy
+        update_filters:
+          - class: stomp_moveit/PolynomialSmoother
+            poly_order: 6
+          - class: stomp_moveit/TrajectoryVisualization
+            line_width: 0.05
+            rgb: [0, 191, 255]
+            error_rgb: [255, 0, 0]
+            publish_intermediate: True
+            marker_topic: stomp_trajectory
+            marker_namespace: optimized      
+    
+   **>** *Save this file in the* **config** *directory of the moveit_config package*. Also make sure that the dimensionality of the `stddev` array parameter is the same as the number of joints present in the planning group name of your robot.
+
+#. Modify the **move_group.launch** file. Open the **move_group.launch** in the launch directory and change the ```pipeline``` parameter value to ```stomp``` as shown below: ::
+
+        .
+        .
+        .
+    <!-- move_group settings -->
+    <arg name="allow_trajectory_execution" default="true"/>
+    <arg name="fake_execution" default="false"/>
+    <arg name="max_safe_path_cost" default="1"/>
+    <arg name="jiggle_fraction" default="0.05" />
+    <arg name="publish_monitored_planning_scene" default="true"/>
+
+    <!-- Planning Functionality -->
+    <include ns="move_group" file="$(find myworkcell_moveit_config)/launch/planning_pipeline.launch.xml">
+      <arg name="pipeline" value="stomp" />
+    </include>
+
+        .
+        .
+        .
+   
+
+Running the Demo
+----------------
+If you have the ``panda_moveit_config`` from the `ros-planning/panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_ repository you should be able to simply run the demo: ::
+
+  roslaunch panda_moveit_config demo.launch
+
+Running STOMP with Obstacles in the Scene
++++++++++++++++++++++++++++++++++++++++++
+To run STOMP in an evironment with obstacles, you can run the sample python script:
```

Comment:
*Python

should these scripts be located in a different folder than ``chomp_planner``? seems they are not chomp specific but just example obstacle scenes

Reply:
could you point me to the directory where I should store this [python script](https://github.com/ros-planning/moveit_tutorials/blob/kinetic-devel/doc/chomp_planner/scripts/collision_scene_example.py)? Should I put the script [here](https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/_scripts) @davetcoleman  ?

Reply:
Can you create a new folder in ``https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc`` called ``collision_environments`` that contains these please?

Reply:
sure, done